src/Toolkit/Collection.php

```diff
@@ -1135,14 +1135,18 @@ public function toString(): string
     }
 
     /**
-     * Returns an non-associative array
-     * with all values
+     * Returns a non-associative array
+     * with all values. If a mapping
+     * Closure is passed, all values
+     * are processed by the Closure.
```

Comment:
```suggestion
     * are processed by the Closure.
     * @since 3.6.0 This also accepts callback parameter to mapping
```

Using more `@since` tags, we can read the code better in the future.

Reply:
I've added it to the description.